packages/desktop-client/src/components/mobile/MobileAmountInput.js

```diff
@@ -0,0 +1,313 @@
+        text: '',
+        ...this.getInitialValue(),
+      });
+    }
+  }
+
+  parseText() {
+    return toRelaxedNumber(
+      this.state.text.replace(/[,.]/, getNumberFormat().separator),
+    );
+  }
+
+  // animate() {
+  //   this.animation = Animated.sequence([
+  //     Animated.timing(this.backgroundValue, {
+  //       toValue: 1,
+  //       duration: 1200,
+  //       useNativeDriver: true,
+  //     }),
+  //     Animated.timing(this.backgroundValue, {
+  //       toValue: 0,
+  //       duration: 1200,
+  //       useNativeDriver: true,
+  //     }),
+  //   ]);
+
+  //   this.animation.start(({ finished }) => {
+  //     if (finished) {
+  //       this.animate();
+  //     }
+  //   });
+  // }
+
+  onKeyPress = e => {
+    if (e.nativeEvent.key === 'Backspace' && this.state.text === '') {
+      this.setState({ editing: true });
+    }
+  };
+
+  getInitialValue() {
+    return {
+      value: Math.abs(this.props.value),
+    };
+  }
+
+  focus() {
+    this.input.focus();
+
+    const initialState = this.getInitialValue();
+    this.setState(initialState);
+  }
+
+  applyText = () => {
+    const { editing } = this.state;
+
+    const parsed = this.parseText();
+    const newValue = editing ? parsed : getValue(this.state);
+
+    this.setState({
+      value: Math.abs(newValue),
+      editing: false,
+      text: '',
+    });
+
+    return newValue;
+  };
+
+  onBlur = () => {
+    const value = this.applyText();
+    this.props.onBlur && this.props.onBlur(value);
+    if (this.removeListeners) {
+      this.removeListeners();
+    }
+  };
+
+  onChangeText = text => {
+    let { onChange } = this.props;
+
+    this.setState({ text });
+    onChange(text);
+  };
+
+  render() {
+    const { style, textStyle } = this.props;
+    const { editing, value, text } = this.state;
+
+    let input = (
+      <input
+        type="text"
+        ref={el => (this.input = el)}
+        value={text}
+        inputMode="decimal"
+        autoCapitalize="none"
+        onChange={e => this.onChangeText(e.target.value)}
+        onBlur={this.onBlur}
+        onKeyPress={this.onKeyPress}
+        data-testid="amount-input"
+        style={{ flex: 1, textAlign: 'center', position: 'absolute' }}
+      />
+    );
+
+    return (
+      <View
+        style={[
+          {
+            justifyContent: 'center',
+            borderWidth: 1,
+            borderColor: colors.p3,
+            borderRadius: 4,
+            padding: 5,
+            backgroundColor: 'white',
+          },
+          style,
+        ]}
+      >
+        <View style={{ overflowY: 'auto' }}>{input}</View>
+
+        {/* <Animated.View
+          style={{
+            position: 'absolute',
+            left: 0,
+            right: 0,
+            bottom: 0,
+            top: 0,
+
+            backgroundColor: animationColor || colors.p10,
+            opacity: this.backgroundValue,
+            borderRadius: 2,
+          }}
+          pointerEvents="none"
+        /> */}
```

Comment:
I could see cleaning these up, but I kinda like seeing what used to be animated in RN to maybe make it easier to add animations on web later.